inc/hclib-place.h

```diff
@@ -87,32 +79,9 @@ extern void *hclib_allocate_at(place_t *pl, size_t nbytes, int flags);
  */
 extern void hclib_free_at(place_t *pl, void *ptr);
 
-#ifdef HC_CUDA
-
-extern place_t **hclib_get_nvgpu_places(int *n_nvgpu_places);
-extern hclib_promise_t *hclib_async_copy(place_t *dst_pl, void *dst,
-        place_t *src_pl, void *src, size_t nbytes,
-        hclib_future_t **future_list, void *user_arg);
-extern void hclib_async_copy_helper(place_t *dst_pl, void *dst, place_t *src_pl,
-        void *src, size_t nbytes, hclib_future_t **future_list,
-        void *user_arg, hclib_promise_t *out_promise);
-extern hclib_promise_t *hclib_async_memset(place_t *pl, void *ptr, int val,
-        size_t nbytes, hclib_future_t **future_list, void *user_arg);
-extern void hclib_async_memset_helper(place_t *pl, void *ptr, int val,
-        size_t nbytes, hclib_future_t **future_list, void *user_arg,
-        hclib_promise_t *out_promise);
-#endif
-
 inline short is_cpu_place(place_t * pl) {
```

Comment:
Is this used anywhere today? Seems like with these changes it would always return true, so maybe it can be deleted too?